eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java

```diff
@@ -2,10 +2,12 @@
 
 import io.eventuate.tram.commands.common.CommandNameMapping;
 import io.eventuate.tram.commands.common.DefaultCommandNameMapping;
+import io.micronaut.context.annotation.Factory;
 import io.micronaut.context.annotation.Requires;
 
 import javax.inject.Singleton;
 
+@Factory
 public class TramCommandCommonFactory {
```

Comment:
What about a test for this? This is.a.bug, right?

Reply:
Without factory annotation, it does not create bean.
Do we really need to test this?

Reply:
* There's some functionality in Tram Sagas that was broken by a bug in Tram.
* Yet the tests for Tram passed.
* Sounds like there is a missing test.

Reply:
ok, I will add